usaspending_api/references/v2/views/award_types.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from usaspending_api.awards.v2.lookups.lookups import (
+    contract_type_mapping,
+    direct_payment_type_mapping,
+    grant_type_mapping,
+    idv_type_mapping,
+    loan_type_mapping,
+    other_type_mapping
+)
+
+
+class AwardTypeGroups(APIView):
+    """
+    This route returns a JSON object describing the award type groupings.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/references/award_types.md"
+
+    def get(self, request, format=None):
+        return Response({
```

Comment:
Wouldn't it be simpler to construct the response by combining the `assistance_type_mapping` and `procurement_type_mapping` dicts?

Reply:
I don't think so; we want the response to map at a more granular level than just `assistance` and `procurement`.

Here's the response object we want from the ticket itself:

```json
{
    "contracts": {
        "A": "BPA Call",
        "B": "Purchase Order",
        "C": "Delivery Order",
        "D": "Definitive Contract"
    },
    "loans": {
        "07": "Direct Loan",
        "08": "Guaranteed/Insured Loan"
    },
    "idvs": {
        "IDV_A": "GWAC Government Wide Acquisition Contract",
        "IDV_B": "IDC Multi-Agency Contract, Other Indefinite Delivery Contract",
        "IDV_B_A": "IDC Indefinite Delivery Contract / Requirements",
        "IDV_B_B": "IDC Indefinite Delivery Contract / Indefinite Quantity",
        "IDV_B_C": "IDC Indefinite Delivery Contract / Definite Quantity",
        "IDV_C": "FSS Federal Supply Schedule",
        "IDV_D": "BOA Basic Ordering Agreement",
        "IDV_E": "BPA Blanket Purchase Agreement"
    },
    "grants": {
        "02": "Block Grant",
        "03": "Formula Grant",
        "04": "Project Grant",
        "05": "Cooperative Agreement"
    },
    "other_financial_assistance": {
        "09": "Insurance",
        "11": "Other Financial Assistance"
    },
    "direct_payments": {
        "06": "Direct Payment for Specified Use",
        "10": "Direct Payment with Unrestricted Use"
    }
}
```